### PR TITLE
GEOMESA-633 fixing time zone of delimited exports

### PR DESCRIPTION
**Pull request by elahrvivaz**

### CODE REVIEW

File: geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/FeatureExporterTest.scala [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      val writer = new StringWriter()
+      val export = new DelimitedExport(writer, Formats.CSV, Some("name,geom,dtg"), None, None, None, None)
+      export.write(featureCollection)
+      export.close()
+
+      val result = writer.toString.split("\n")
+      val (header, data) = (result(0), result(1))
+
+      header mustEqual "name,geom,dtg"
+      data mustEqual "myname,POINT (45 49),1970-01-01 00:00:00"
```
Comment (jnh5y) [inline geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/FeatureExporterTest.scala]: Does this test fail without the fix in FeatureExporter.scala?

Comment (elahrvivaz) [inline geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/FeatureExporterTest.scala]: Yeah, the date gets formatted as 1969-12-31 16:00:00 or something like that.

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/FeatureExporter.scala [hunk 2]
```diff
@@ -119,8 +119,13 @@ class DelimitedExport(writer: Writer,
    case Formats.CSV => ","
    case Formats.TSV => "\t"
   }
-  lazy val dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+
   lazy val geometryFactory = JTSFactoryFinder.getGeometryFactory
+  lazy val dateFormat = {
+    val df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
```
Comment (jahhulbert-ccri) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/FeatureExporter.scala]: is this the default for all exports? If so can we make it full precision 8601 yyyy-MM-dd'T'HH:mm:ss.SSSZ or is the other format standard for geotools stuff?

**Comment by jahhulbert-ccri** [comment]
looks good to me

**Comment by jnh5y** [comment]
Committed as 2d95a759c686ae147c46ef2786c84afbe40cfda1 on 1.5.

**Pull request closed without merging**